ang/civicase-base/providers/case-type-category.provider.js

```diff
@@ -29,11 +33,55 @@
     function $get () {
       return {
         getAll: getAll,
+        findById: findById,
         findByName: findByName,
-        getCategoriesWithAccessToActivity: getCategoriesWithAccessToActivity
+        findAllByInstance: findAllByInstance,
+        getCategoriesWithAccessToActivity: getCategoriesWithAccessToActivity,
+        isInstance: isInstance
       };
     }
 
+    /**
+     * Find all case type categories belonging to sent instance name
+     *
+     * @param {string} instanceName name of the instance
+     * @returns {Array} list of case type categories matching the sent instance
```

Comment:
can we use `object[]` instead of `Array`? `Array` can contain any type of data, but `object[]` means that it's an array of objects.